src/assets/styles/base.scss

```diff
@@ -259,15 +259,11 @@ ul.list {
 
 .no-results__subtitle {
   margin-top: 0.25rem;
-  color: $gray6;
+  color: $gray7;
   @extend .font-14;
   font-weight: 500;
 }
 
-.gray6 {
-  color: $gray6;
-}
```

Comment:
This class was only used in `Dashboard.vue` for the count numbers. I created a custom css class in `Dashboard.vue` for these numbers so this class is no longer needed.

Reply:
That's fine.

We float back and forth on this. If it's used in just 1 place then declare it there, otherwise it would be nice to have a "library" of classes for setting color, or text size, or whatever (future, if needed).